bin/secrets

```diff
@@ -6,13 +6,20 @@ vault_auth_pem = ENV["VAULT_AUTH_PEM"] || '/vault-auth/pem'
 vault_tls_verify = ['1', 'true'].include?(ENV["VAULT_TLS_VERIFY"])
 sidecar_secret_path = ENV["SIDECAR_SECRET_PATH"] || '/secrets'
 annotations = ENV["SECRET_ANNOTATIONS"] || '/secretkeys/annotations'
+serviceaccount_dir = ENV["SERVICEACCOUNT_DIR"] || '/var/run/secrets/kubernetes.io/serviceaccount/'
+api_host = ENV.fetch("KUBERNETES_PORT_443_TCP_ADDR")
```

Comment:
nope, the options are:
KUBERNETES_PORT=tcp://10.247.0.1:443
KUBERNETES_PORT_443_TCP_PORT=443
KUBERNETES_SERVICE_PORT=443
KUBERNETES_SERVICE_HOST=10.247.0.1
KUBERNETES_PORT_443_TCP_PROTO=tcp
KUBERNETES_SERVICE_PORT_HTTPS=443
KUBERNETES_PORT_443_TCP_ADDR=10.247.0.1
KUBERNETES_PORT_443_TCP=tcp://10.247.0.1:443

so I could switch it to KUBERNETES_SERVICE_HOST if you'd prefer.